tutorial/11-testing-mocha-chai-sinon/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Mocha and Chai
 
-- Create an `src/test` folder. This folder will mirror our application folder structure, so create a `src/test/client` folder as well (feel free to add `server` and `shared` if you want, but we're not going to write tests for these).
+- Create a `src/test` folder. This folder will mirror our application folder structure, so create a `src/test/client` folder as well (feel free to add `server` and `shared` if you want, but we're not going to write tests for these).
```

Comment:
This one is not so obvious. It depends on how you pronounce "src". I read it "ess ar cee", not "source", and in this case, it should remain "an".